software/base/src/main/java/org/apache/brooklyn/entity/software/base/AbstractSoftwareProcessWinRmDriver.java

```diff
@@ -154,11 +214,11 @@ protected WinRmToolResponse executeCommand(ConfigKey<String> regularCommandKey,
         Task<?> currentTask = Tasks.current();
         if (currentTask != null) {
             writeToStream(stdIn, Strings.isBlank(regularCommand) ? powershellCommand : regularCommand);
-            Tasks.addTagDynamically(BrooklynTaskTags.tagForStreamSoft(BrooklynTaskTags.STREAM_STDIN, stdIn));
+            Tasks.addTagDynamically(BrooklynTaskTags.tagForStreamSoft(BrooklynTaskTags.STREAM_STDIN, stdIn)); flags.put("stdin", stdIn);
 
             if (BrooklynTaskTags.stream(currentTask, BrooklynTaskTags.STREAM_STDOUT)==null) {
-                Tasks.addTagDynamically(BrooklynTaskTags.tagForStreamSoft(BrooklynTaskTags.STREAM_STDOUT, stdOut));
-                Tasks.addTagDynamically(BrooklynTaskTags.tagForStreamSoft(BrooklynTaskTags.STREAM_STDERR, stdErr));
+                Tasks.addTagDynamically(BrooklynTaskTags.tagForStreamSoft(BrooklynTaskTags.STREAM_STDOUT, stdOut)); flags.put("stdout", stdOut);
```

Comment:
Can you separate these commands onto multiple lines - it risks being missed by someone reading the code when at the end of a long line like this. Same for the code above.